### DISCUSSION TITLE
ScriptReference/Application-targetFrameRate

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#249] [Notes]
# ScriptReference/Application-targetFrameRate

https://docs.unity3d.com/ScriptReference/Application-targetFrameRate.html

**Comment by radiatoryang** [comment]
For desktop, it's highly recommended to *always* set this to something like 60 or 120, just to clamp the framerate -- otherwise systems running at very high framerates without vsync can shoot up to like 1000 FPS and cause weird bugs or errors.
